codespell_lib/data/dictionary.txt

```diff
@@ -23865,6 +23882,7 @@ elelmentary->elementary
 elelments->elements
 elemant->element
 elemantary->elementary
+elemants->elements
```

Comment:
I don't suppose elephants should be added here as a possible fix?

Reply:
I think that's a stretch